charts/meteor-pipelines/templates/byon-validate-jupyterhub-image.yaml

```diff
@@ -10,12 +10,12 @@ spec:
   workspaces:
     - name: data
   steps:
-    - name: jupyter-book-build
+    - name: get-package-versions
       image: $(params.url)
       script: |
         #!/usr/bin/env bash
-        # Check Python versionand
-        echo  "Validating image:"
+        # Check Python version
+        echo  "Analyzing image:"
```

Comment:
The changes to the Task  come from https://github.com/thoth-station/helm-charts/pull/34, right?

I believe this file was included in this PR by mistake?

Reply:
No, this PR depends on #34, hence it includes the commit from the #34 PR. Once #34 is merged, this PR will become a single commit PR ->  https://github.com/thoth-station/helm-charts/pull/35/commits/41b75abfa58ea1486696255051a23e91160aadbe